src/pushInfo/PushInfo.story.mdx

```diff
@@ -0,0 +1,42 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+
+import { ProximityIcon } from '../icon/proximityIcon'
+import { PushInfo } from './index'
+
+<Meta title="Widgets|PushInfo" />
```

Comment:
What do you think to create an "Alert" category?

Reply:
What else would you put in there?

Reply:
"Alert" elements like `Snackbar`

Reply:
I'm not sure it qualifies as an alert, but either way, I'd rather to keep the category out of the MDX migration for now.